acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -85,6 +88,15 @@ public void waitForNewBlock() throws IOException {
         () -> assertThat(getCurrentBeaconHead().getBlockRoot()).isNotEqualTo(startingBlockRoot));
   }
 
+  public void waitForNewFinalization() throws IOException {
+    UnsignedLong startingFinalizedEpoch = getCurrentFinalizedCheckpoint().getEpoch();
+    waitFor(
+        () ->
+            assertThat(getCurrentFinalizedCheckpoint().getEpoch())
+                .isNotEqualTo(startingFinalizedEpoch),
+        2000);
```

Comment:
2000 seconds is probably too long to be waiting. 1-2 minutes at most would make sense - it only needs a relatively small buffer over the time required for that number of slots to progress.

As a post-1.0 thing, we should add the ability to time travel to these tests - basically just tell Artemis to 'fast forward' time so we don't actually have to wait for the slots to progress.  That will require a bunch of work so definitely not for now.

Reply:
waiting for finality actually takes about 7 minutes. I can make it a tighter wait time like 9 minutes which would be 540 seconds.